src/pages/home.jsx

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react'
+
+class Home extends Component{
+    render(){
+        return(
+            <div>
+                
+                <div className="logo-title">
+                    <img src="logo.png"/>
+                    <p className="app-title">ChitChatter</p>
+                </div>
+
+                <div className="container">
+                    <form>
+                        <div className="form-inputs">
+                            <input type="text" name="username" placeholder="username"/>
+                        </div>
+                        <div className="form-inputs">
+                            <input type="text" name="chatroomname" placeholder="chatroom-name"/>
+                        </div>
+                        <input type="submit" name="submit" value="Enter Room"/>
```

Comment:
Can you add an `onClick` handler here that calls this function 
`this.props.history.push('/chatroom')`

This will change the page to /chatroom when they click login

Reply:
you may have to call `e.preventDefault()` to stop the page from refreshing